code/src/index.css

```diff
@@ -1,13 +1,170 @@
+:root {
+  accent-color: rgb(206, 13, 13);
+  --primary: rgb(68, 13, 36);
+  --red: rgba(255, 44, 44, 0.473);
+  --white: rgb(255, 255, 255);
+}
+
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+  font-family: -apple-system, Montserrat, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  box-sizing: border-box;
+  background: url(./assets/alexandru-acea-RQgKM1h2agA-unsplash.jpg);
```

Comment:
I think my only gripe with the styling is that the red color text for the summary is a bit difficult to read with the opacity and the background image - that's the only thing I can point a finger at!